src/Internal/FilesystemHelper.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Internal;
+
+use function clearstatcache;
+use function is_dir;
+use function mkdir;
+
+final class FilesystemHelper
```

Comment:
Please name this class `DirectoryCreator` as I asked here: https://github.com/phpstan/phpstan-src/pull/2697#issuecomment-1784024859

Reply:
Done, I used `FilesystemHelper` because